cost-analyzer/values-eks-cost-monitoring.yaml

```diff
@@ -23,4 +23,20 @@ prometheus:
   configmapReload:
     prometheus:
       image:
-        repository: public.ecr.aws/kubecost/prometheus-config-reloader
\ No newline at end of file
+        repository: public.ecr.aws/kubecost/prometheus-config-reloader
+
+reporting:
+  productAnalytics: false
+
+# grafana is disabled by default, but can be enabled by setting the following values.
+# or proxy to an existing grafana: https://docs.kubecost.com/install-and-configure/advanced-configuration/custom-grafana
+global:
+  grafana:
+    enabled: false
+    proxy: false
+# grafana:
+#   image:
+#     repository: YOUR_REGISTRY/grafana
+#   sidecar:
+#     image:
+#       repository: YOUR_REGISTRY/k8s-sidecar
```

Comment:
Do we need these?

Reply:
just trying to make it obvious how to use a different registry for grafana. I could take it or leave it.

Reply:
We can push those images. It's a trivial matter and the automation work is done. Should we do that?